pipelines/pr.yaml

```diff
@@ -22,5 +22,5 @@ jobs:
       # the ongoing rolling CI build. This build in particular is highly correlated
       # the .NET x86 build and ARM IL2CPP build.
       buildUWPX86: false
-      buildUWPDotNet: false
+      buildUWPDotNet: true

```

Comment:
This can just be removed, since it defaults to `true` in the template

Reply:
sounds good